lib/services/elasticsearch.js

```diff
@@ -74,7 +69,11 @@ class ElasticSearch extends Service {
     }
 
     if (process.env.NODE_ENV === 'production' && this.config.dynamic === 'true') {
-      this.kuzzle.log.warn('Your dynamic mapping policy is set to \'true\' for new fields.\nElasticsearch will try to automatically infer mapping for new fields, and those cannot be changed afterward.\nSee the "services.db.dynamic" option in the kuzzlerc configuration file to change this value.'
+      this.kuzzle.log.warn('Your dynamic mapping policy is set to \'true\' ' +
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this concatenation to the use of a template. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3512)